os&sys/wc.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+
+import sys
+if len(sys.argv) == 1:
+    print("Usage: {} FILENAMEs".format(sys.argv[0]))
+else:
+    if sys.argv[1] == "l" or sys.argv[1] == "w" or sys.argv[1] == "c":
+        filenames = sys.argv[2:]
+    else:
+        filenames = sys.argv[1:]
+    all_lines = 0
+    all_words = 0
+    all_chars = 0
+    for filename in filenames:
+        lines = 0
+        words = 0
+        chars = 0
+        with open(filename) as file:
+            for line in file:
+                lines += 1
+                words += len(line.split())
+                chars += len(line)
+        if sys.argv[1] == "l":
+            print(lines, filename)
+        elif sys.argv[1] == "w":
+            print(words, filename)
+        elif sys.argv[1] == "c":
+            print(chars, filename)
```

Comment:
```suggestion
        if sys.argv[1] == "-l":
            print(lines, filename)
        elif sys.argv[1] == "-w":
            print(words, filename)
        elif sys.argv[1] == "-c":
            print(chars, filename)
```
Это совершенно не критично, но обычно перед такими флагами указывают минус